src/schema/objects/metadata.yaml

```diff
@@ -3392,6 +3392,24 @@ SubjectArtefactDescription:
     If this field is set to `"n/a"`, it will be interpreted as absence of major
     source of artifacts except cardiac and blinks.
   type: string
+TablePosition:
+  name: TablePosition
+  display_name: Table Position
+  description: |
+    The table position, relative to an implementation-specific reference point,
+    often the isocenter. Values must be an array (1x3) of three distances in
+    millimeters in absolute coordinates (world coordinates). The
+    absolute coordinates are respectively right, anterior and superior (RAS+)
+    as if it was acquired in head first supine (HFS) orientation. In
+    other words, moving the table into the scanner will always increase the 3rd
+    value in the array, regardless of the patient positioning (feet
```

Comment:
I'm wondering if we should just avoid mentioning examples of patient positioning, because in the end TablePosition is independent of patient positioning. The way you defined it is good enough I think (ie: table goes in the scanner means S increases). Adding refs to patient positioning adds confusion I find.